README.md

```diff
@@ -88,7 +88,7 @@ api.getAnalyticDoc(analyticId).then(function(doc) {
 
 ### Data
 
-Upload data to the cloud:
+Upload data to the cloud. pass path to the file along with file name as an argument:
```

Comment:
"Upload data to cloud storage, where `uploadDataPath` is the absolute or relative path to the local date file."